docs/Changelog.md

```diff
@@ -5668,9 +5668,9 @@ This version of the operator has been available since version 6 of the default O
 #### Attributes
 
 <dl>
-<dt><tt>max</tt> : float (default is 3.4028234663852886e+38)</dt>
+<dt><tt>max</tt> : float (default is 3.40282346639e+38)</dt>
```

Comment:
this change is not you want :), I think.

Reply:
Any idea what causes this? (I guess something in the document generator pipeline might be using lower precision, but not sure where.)